tools/hardware/utils/memory.py

```diff
@@ -57,6 +89,7 @@ def reserve_regions(regions: Set[Region], reserved: Set[Region]) -> Set[Region]:
 def get_physical_memory(tree: FdtParser, config: Config) -> List[Region]:
     ''' returns a list of regions representing physical memory as used by the kernel '''
     regions = get_memory_regions(tree)
+    regions = merge_memory_regions(regions)
```

Comment:
let's merge the lines
```suggestion
    regions = merge_memory_regions(get_memory_regions(tree))
```

Reply:
Done.